app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationRouter.kt

```diff
@@ -46,11 +42,16 @@ import com.wire.kalium.logic.data.user.UserId
 @Composable
 fun ConversationRouterHomeBridge(
     onHomeBottomSheetContentChanged: (@Composable ColumnScope.() -> Unit) -> Unit,
-    onBottomSheetVisibilityChanged: () -> Unit,
+    openBottomSheet: () -> Unit,
+    setSnackBarState: (HomeSnackbarState) -> Unit,
     onScrollPositionProviderChanged: (() -> Int) -> Unit
 ) {
     val viewModel: ConversationListViewModel = hiltViewModel()
 
+    LaunchedEffect(Unit) {
+        viewModel.snackBarState.collect { setSnackBarState(it) }
```

Comment:
it would be better to do it like this : 

LaunchedEffect(snackbarState){
setSnackBarState(snackbarState)
}

Reply:
but `viewModel.snackBarState` is a Flow, not a State. 
I used `Flow` here to not show snackbar every time `ConversationRouterHomeBridge` recomposed.